src/makeRobotAccountant.js

```diff
@@ -6,7 +6,17 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let count = 0;
+
+  return a => {
+    return b => {
```

Comment:
make sure to add parentheses even if you have only one param, it improves code readability
```suggestion
  return (a) => {
    return (b) => {
```